src/Calypso-SystemPlugins-SUnit-Browser/ClyTestCommand.class.st

```diff
@@ -31,14 +31,15 @@ ClyTestCommand >> defaultMenuIcon [
 { #category : #execution }
 ClyTestCommand >> notifyUserAboutResults: testResult with: message [
 
-	| color |	
+	| color theme |	
 	color := Color gray.
+	theme := Smalltalk ui theme.
 	testResult hasPassed
-		ifTrue: [ color := Color green ].
+		ifTrue: [ color := theme successBackgroundColor ].
 	testResult hasFailures
-		ifTrue: [ color := Color yellow ].
+		ifTrue: [ color := theme warningBackgroundColor ].
 	testResult hasErrors
-		ifTrue: [ color := Color red ].
+		ifTrue: [ color := theme dangerBackgroundColor].
```

Comment:
I suggest to follow SUnit vocabulary: greenTestColor, yellowTestColor and redTestColor. This is well established naming.

Reply:
I like this. We wouldn't impact other users.
But should the other users keep those colors? 
It's a real question I'm wondering, I don't know about everything that's affected, but if it's as unreadable as miniDrTest, it might make sense to still change them.